website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -5,22 +5,20 @@ weight: 300
 description: Instructions for migrating a legacy Helm-based project to use the new Kubebuilder-style layout.
 ---
 
-This guide walks through an example of migrating a simple nginx-operator which was built by following the [legacy quick-start][quickstart-legacy] to the new layout.
-
 ## Overview
 
 The motivations for the new layout are related to bringing more flexibility to users and 
 part of the process to [Integrating Kubebuilder and Operator SDK][integration-doc].
 
 ### What was changed
  
-The `deploy` directory was replaced with the `config` directory including a new layout of Kubernetes manifests files
-- CRD's manifests in `deploy/crds/` are now in `config/crd/bases`
-- CR's manifests in `deploy/crds/` are now in `config/samples`
-- Controller manifest `deploy/operator.yaml` was replaced for `config/manager/manager.yaml` 
-- RBCA's manifests in `deploy` are in `config/rbac/`
+The `deploy` directory was replaced with the `config` directory including a new layout of Kubernetes manifests files:
+- CRD manifests in `deploy/crds/` are now in `config/crd/bases`
+- CR manifests in `deploy/crds/` are now in `config/samples`
+- Controller manifest `deploy/operator.yaml` is now in `config/manager/manager.yaml` 
+- RBAC manifests in `deploy` are now in `config/rbac/`
 
-The `build/Dockerfile` directory was replaced by the `Dockerfile` in the root directory
+`build/Dockerfile` is moved to `Dockerfile` in the project root directory
```

Comment:
Move this to the bullet list.

Reply:
it is not in the bullet: 👍 
<img width="767" alt="Screenshot 2020-07-30 at 14 20 16" src="https://user-images.githubusercontent.com/7708031/88927789-dbdb3300-d26f-11ea-9647-b663c1919f56.png">

Reply:
Done as:

<img width="881" alt="Screenshot 2020-07-30 at 14 41 50" src="https://user-images.githubusercontent.com/7708031/88943459-6aa57b00-d283-11ea-96ec-8f56cbdff4c9.png">